### PR TITLE
Added: Guest blogging opportunity at Logical Clocks

### PR DESCRIPTION
**Pull request by balapriyac**
I've added the Guest Bloggers program at Logical Clocks to the list. The writers can submit code demos and tech tutorials on the areas of Data Engineering, Machine Learning, and ML Engineering.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,9 @@ A list of companies that have paid Developer Community Writer Programs.
 
 - [Linode](https://www.linode.com/docs/contribute/) - Up to $300 per piece
   > Technical tutorials with code on Linux or Linode.
+  
+- [Logical Clocks](https://www.logicalclocks.com/job-positions/guest-blogger) - Pay-per-tutorial setting, GPU credits
```
Comment (malgamves) [inline README.md:101]: hey, is there a monetary value assigned to the compensation? if yes, could you add that? We only add publications that pay writers to the list

**Review by malgamves — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (balapriyac) [reply README.md:101]: Hi, I'm not sure of the exact compensation they provide.
The following information is available on the website:
### Benefits

- Compensation for each blog published;
- build your brand by publishing at a world leading ML company, including your biography and photo on each blog you write;
- get free access to lots of GPUs on the Hopsworks platform.

Comment (karllhughes) [reply README.md:101]: @balapriyac any word on how much they compensate writers?

We'd prefer to only include sources with set payment rates. Thanks!

**Comment by karllhughes** [comment]
@balapriyac if you could add their pay rate, we can go ahead and merge this in. If not, we'll close this PR as we prefer to only include programs with publicly known pay rates. Thanks!

File: README.md (hunk 1 above)
Comment (balapriyac) [reply README.md:101]: Hi @karllhughes, 
I'm sorry. I couldn't find their pay per article. Kindly close the PR.

**Pull request closed without merging**